README.rst

```diff
@@ -66,6 +66,35 @@ Features
 * Should work with Python 3.6 and newer. (Versions 2019.2 and older continue
   to support Python 2.7.)
 
+Configuration
```

Comment:
How come this is in the README and not in the documentation (under `doc/`)?

Reply:
@inducer Not sure If I understood correctly. but I dont see README.rst under doc/ in the main branch.

Reply:
My point is that I think the documentation (see rendered at https://documen.tician.de/pudb/, generated from the files under `doc/` via sphinx) is likely a better spot for this than the README (at least IMO).

Reply:
okay. I will take a look at how it could be added and generated. 
We can merge this now if it is okay and track these doc changes through a subsequent PR in the meanwhile I figure out generation of docs. Either of these works. Let me know